app-crypt/tpm2-pkcs11/tpm2-pkcs11-1.2.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_REQ_USE="sqlite"
+
+inherit distutils-r1
+
+DESCRIPTION="A PKCS#11 interface for TPM2 hardware"
+HOMEPAGE="https://tpm2-software.github.io/"
+SRC_URI="https://github.com/tpm2-software/tpm2-pkcs11/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+# Test requires the IBM TPM simulator that's not in portage
+RESTRICT="test"
+
+RDEPEND="app-crypt/p11-kit
+	>=app-crypt/tpm2-abrmd-2.1.0
+	>=app-crypt/tpm2-tss-2.3.0
+	dev-db/sqlite
+	>=dev-libs/openssl-1.0.2
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	dev-python/cryptography[${PYTHON_USEDEP}]
+	dev-python/pyasn1[${PYTHON_USEDEP}]
+	dev-python/pyasn1-modules[${PYTHON_USEDEP}]"
+
+# Build-time dependencies that need to be binary compatible with the system
+# being built (CHOST). These include libraries that we link against.
+# The below is valid if the same run-time depends are required to compile.
+DEPEND="${RDEPEND}"
+
+# Build-time dependencies that are executed during the emerge process, and
+# only need to be present in the native build system (CBUILD). Example:
+BDEPEND=">=sys-devel/autoconf-archive-2018.03.13
+	virtual/pkgconfig"
+
+REQUIRED_USE="( ${PYTHON_REQUIRED_USE} )"
+
+src_configure()
+{
+	econf
+}
+
+src_compile()
+{
+	emake
+	cd "${S}/tools" && BUILD_DIR="${S}/tools" python_foreach_impl distutils-r1_python_compile
+}
+
+src_install()
+{
+	emake DESTDIR="${D}" install
+	einstalldocs
+	cd "${S}/tools" && BUILD_DIR="${S}/tools" python_foreach_impl distutils-r1_python_install
```

Comment:
Add `|| die` and multi-line this properly.

Reply:
Added and split